src/common/redis_module/ray_redis_module.cc

```diff
@@ -6,6 +6,8 @@
 #include "format/common_generated.h"
 #include "task.h"
 
+#include "../../ray/gcs/format/gcs_generated.h"
```

Comment:
Relative paths in includes are a red herring. :-)  Can we do "format/gcs_generated.h"?

Reply:
fixed (also all new include files are prefixed by ray/ now)